worlds/dkc2/Tracker.py

```diff
@@ -0,0 +1,1275 @@
+from typing import Any, TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from . import DKC2World
+
+def map_page_index(data: Any) -> int:
+    mapping: dict[str, int] = {
+        0x100: 0, # World Map
```

Comment:
The type here doesn't match the data. Also I'm not sure how the tracker ends up using this function, but does it need to take in `Any` when the only type present in the map is `int`? If that gets removed then the import for `Any` could as well.